source/_integrations/proxmoxve.markdown

```diff
@@ -100,9 +101,39 @@ proxmoxve:
 
 ## Binary Sensor
 
-The integration will automatically create a binary sensor for each tracked virtual machine or container. The binary sensor will either be on if the VM's state is running or off if the VM's state is different.
+The integration will automatically create a binary sensor for each tracked virtual machine or container. The binary sensor will either be on if the VM's state is running or off if the VM's state is different. There are als additional attributes added to each binary sensor for the node running on, VM name, VM id, VM type, CPU count and memory size.
+
+The created sensor will be called `binary_sensor.NODE_VMNAME_running`.
+
+example for additional attributes
+```yaml
+node: pve
+vmid: 207
+vmname: devhost
+type: lxc
+cpu: 8
+memory: 16384
+```
+
+## Sensor
+
+The integration will automatically create multiple sensors for each tracked virtual machine or container about their cpu, memory and disk utilisation.
+
+The created sensors will be called `sensor.NODE_VMNAME_SENSORNAME`.
```

Comment:
This line should not be needed, as they can be found via the devices and customized in the UI.